### DISCUSSION TITLE
Is there a way to actively see what the backend server is doing through the web UI?

### DISCUSSION THREAD

**Discussion by TheGorf** [immich-app/immich#7718] [Q&A] [answered]
I wanna make sure I'm not missing the obvious here hah. After a lot of testing, I decided to upload a huge volume of pics and movies and try it at scale. All looks great, but my little server is running hard for the last few hours. The docker container is clearly doing "something" but it's hard to tell what. Thumbnails? Movie scaling? AI? I'm just curious how I could know.

**Comment by alextran1502** [comment, marked as answer]
You can go to the Administration page on the web, then to go the Job section to see all the tasks are being processed

**Reply by TheGorf** [reply]
Oh god see I knew it was right in front of me. Thanks!
